client/src/components/QA/QuestionItem.jsx

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable react/prop-types */
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import AnswersList from './AnswersList.jsx';
+
+const Wrapper = styled.section`
+height: 100%;
+background: white;
+margin: 40px;
+`;
+
+const Row = styled.div`
+    display: flex;
+    align-items: center;
+`;
+
+const P = styled.p`
+    margin-left: auto;
+`;
+
+const AlignRight = styled.div`
+    margin-right: auto;
+    padding-right: 100px;
+`;
+
+const QuestionItem = ({ question }) => (
+  <Wrapper>
+    <Row>
+      <AlignRight>
+        {' '}
+        <h2>
+          Q:
+          {question.question_body}
+        </h2>
+        {' '}
+      </AlignRight>
+      <P>
+        {' '}
+        helpful? Yes
+        (
+        {question.question_helpfulness}
+        )
+        {' '}
+        | Add Answer
```

Comment:
I would just combine this into one line using backticks. This also makes it easier to add spacing between Yes and the numbers. Make sure to add $ before variables if you change it.